hw/top_earlgrey/syn/asic.constraints.sdc

```diff
@@ -180,7 +180,7 @@ set_output_delay ${SPI_DEV_OUT_DEL} [get_ports SPI_DEV_D3]   -clock SPI_DEV_CLK
 set_false_path -through [get_pins top_earlgrey/u_spi_device/cio_csb_i] \
                -through [get_pins top_earlgrey/u_spi_device/cio_sd_en_o*]
 set_false_path -through [get_pins top_earlgrey/u_spi_device/cio_csb_i] \
-               -through [get_pins top_earlgrey/u_spi_device/cio_sd_o*]
+    -through [get_pins top_earlgrey/u_spi_device/cio_sd_o*]
```

Comment:
Is this false_path ok for all SPI modes?   I think it may be worth reviewing this later to make sure that is the case.

Reply:
@eunchan 
i THINK it's okay. Since for spi_device, `csb` acts as a reset, it doesn't need to satisfy strict timing.
Although come to think of it, we should probably make this more explicit (say multicycle) and spec that as a minimum for our chip on how frequently csb can be de-asserted / asserted.

Reply:
As far as I know there's no logic path from CSb to host return data. So false_path is fine. 

@cdgori mentioned about this constraint before. Maybe define a buf on CSb and `set_ideal_network` from the buf out would be another solution.

Reply:
ah yes there's no data path inside our chip.
But i mean this more from external perspective.  For example, let's say for whatever reason the CSb reset took 10 cycles to propagate in our design (this is an extreme example), that would imply that our logic internally doesn't reset for that much time, and the outside world should not assert CSb again for at least that many cycles.

So if we can actually constrain it to something, it will help with datasheet writing i think.

Reply:
I agree with trying to constrain it, I just don't know enough about the logic here to suggest how to do it.  :(

The false-path is always a bit worrisome just because it might cover more logic than you want it to, especially if something changes in the code later.